Fix detection of Qt5 if both Qt5 and Qt6 are installed.

--- CMakeLists.txt.orig	2023-07-10 19:51:30 UTC
+++ CMakeLists.txt
@@ -273,7 +273,7 @@ else()
 if(BUILD_WITH_QT6)
   find_package(QT NAMES Qt6 COMPONENTS Core REQUIRED)
 else()
-  find_package(QT NAMES Qt5 Qt6 COMPONENTS Core REQUIRED)
+  find_package(QT NAMES Qt5 COMPONENTS Core REQUIRED)
 endif()
 if(QT_VERSION_MAJOR EQUAL 6 AND QT_VERSION_MINOR LESS 2)
   set(WITH_MULTIMEDIA OFF)
